charms/kserve-controller/requirements-lint.txt

```diff
@@ -2,63 +2,50 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --resolver=backtracking requirements-lint.in
+#    pip-compile ./requirements-lint.in
```

Comment:
I think pip-compile was run manually here, not using the `tox` env?  Renovate will trigger a PR right after this to at least add the `--resolver=backtracking` (and maybe also remove the `./`)

Reply:
It seems like `pip-compile` has now that resolver as default, I ran the command with `--resolver=backtracking`, but pip compile did not add that in the comment in the final .txt file. I'm using this version in case you want to try it yourself:

```
$ pip-compile --version
pip-compile, version 7.1.0
```

Reply:
Do you have an example of renovate triggering a PR if the `./` is left behind?

Reply:
Yep you're right about backtracking

For the `./`, you can see renovate's output [here](https://github.com/canonical/kserve-operators/pull/138/files#diff-58f1ccf94fe863ad70221389f88886b9702a3676d724c76b9474fc102f131846).  Atm I think you have to manually remove the `./` from this line and all the comments in the file (or let renovate open PRs to remove it for you after merging).  Maksim has a fix for our tox.ini [here](https://github.com/canonical/argo-operators/pull/100) that seems to work and remove the annoying `./`